README.md

```diff
@@ -1,11 +1,70 @@
 # DR2 Disaster Recovery
 
-A Lambda
+This is a service which is intended to run in a long-running Docker container.
+
+Every 20 seconds, it polls the queue specified in the `SQS_QUEUE_URL` environment variable.
+This will be set to the queue which receives messages from the entity event generator.
+
+## Messages
+
+The queue sends messages in this format
+
+```json
+{
+  "id": "io:1b9555dd-43b7-4681-9b0d-85ebe951ca02"
+}
+```
+
+The entity type can be `io`, `co` or `so` depending on the entity type. They are dealt with differently.
+
+### Information object messages
```

Comment:
Just for clarity, maybe this title and the CO one could be something be "Handling Information object messages" or "How Information object messages are handled"?

Reply:
Handling Information object messages is ok.